src/community/design-system-day-2022/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Watch videos from Design System Day 2022
+description: Videos, slides and notes from Design System Day 2022.
+section: Community
+theme: Events and workshops
+layout: layout-pane.njk
+order: 12
+---
+
+## Inclusivity by design
+
+Our theme for Design System Day 2022 was inclusivity by design. <!-- Add something about what that means -->
```

Comment:
We've never explained it further, but paragraph could include a general summary of the event.

Reply:
Do we have anything written up that we can re-use, @ImranH-GDS?

Reply:
This can be added after publication.